projects/elife.yaml

```diff
@@ -2118,3 +2118,19 @@ kubernetes-aws:
                 max-size: 3
                 desired-capacity: 3
             helm: true
+
+large-repo-wrangler:
+    description: when elife-publications requires a large git repository to be wrangled. adhoc instances only.
+    domain: False
+    intdomain: False
+    formula-repo: https://github.com/elifesciences/large-repo-wrangler-formula
+    aws:
+        ec2: {}
```

Comment:
wait, why do you need this? It could be omitted as it's inherited?

Reply:
that's a good point, let me test that ...

Reply:
appears to work. I'll change it.